### ISSUE TITLE
Autoplay media is blocking the listen-only audio from playing

### ISSUE THREAD

**Issue by capilkey** [bigbluebutton/bigbluebutton#8354]
Steps to reproduce:
1. Block autoplay media in your browser (I used Firefox)
2. Join a meeting as a moderator
3. Lock the microphone
4. Join the meeting as a viewer
5. The client will automatically try to connect to listen-only audio

Expected result:
The viewer client should connect to listen-only audio and start playing it.

Actual result:
Tons of warnings in the browser about being unable to play media without a user action. Bonus is that the Kurento listen-only fails out and falls back to sip.js listen-only for some reason and that also fails.

Kurento errors
![image](https://user-images.githubusercontent.com/1395090/69369080-5d061300-0c69-11ea-828c-c5fcbfec594f.png)

Sip.js errors
![image](https://user-images.githubusercontent.com/1395090/69369108-714a1000-0c69-11ea-91e1-69f85fbcbf04.png)

**Comment by prlanzarin** [comment]
Error is not quite clear to me. The listen only logs for the stuff that goes through SFU are cut in the middle (or the server couldn't negotiate the listen only stream). The initial warnings in there come from a leftover code that tries to autoplay a fake audio.
There's code for listen-only through SFU that handles autoplay failures. I tested it in Firefox again with a clear cache, cookies and no gUM permissions and it seems to be still working for me.

This is what happens for the SFU LO when an autoplay error is triggered (after the thing is flowing through KMS, see `Start request accepted for audio`):

```
[10:54:33:0541]  INFO: clientLogger: Start request accepted for audio https://etc.bigbluebutton.org/html5client/compatibility/kurento-extension.js?v=798:340:17 d34c3ce949618639a85d6b713df5732685e8b96a.js:9:342294
<autoplay warning for fake audio> A reprodução automática só é permitida quando aprovada pelo usuário, ou o site foi ativado pelo usuário, ou a mídia está com som desativado. d34c3ce949618639a85d6b713df5732685e8b96a.js:125:1154926
<actual autoplay error> [10:54:34:0894] ERROR: clientLogger: Listen only media play failed due to autoplay error https://etc.bigbluebutton.org/html5client/d34c3ce949618639a85d6b713df5732685e8b96a.js?meteor_js_resource=true:125:1154985 d34c3ce949618639a85d6b713df5732685e8b96a.js:9:342294
[10:54:34:0898]  INFO: clientLogger: Prompting user for action to play listen only media https://etc.bigbluebutton.org/html5client/d34c3ce949618639a85d6b713df5732685e8b96a.js?meteor_js_resource=true:125:1139172 d34c3ce949618639a85d6b713df5732685e8b96a.js:9:342294
[10:54:34:0906]  INFO: clientLogger: Audio Joined https://etc.bigbluebutton.org/html5client/d34c3ce949618639a85d6b713df5732685e8b96a.js?meteor_js_resource=true:125:1134940 d34c3ce949618639a85d6b713df5732685e8b96a.js:9:342294
<actual autoplay warning> A reprodução automática só é permitida quando aprovada pelo usuário, ou o site foi ativado pelo usuário, ou a mídia está com som desativado. join
<autoplay handler> [10:56:19:0027]  INFO: clientLogger: Listen only autoplay allowed by the user https://etc.bigbluebutton.org/html5client/d34c3ce949618639a85d6b713df5732685e8b96a.js?meteor_js_resource=true:125:1138623 d34c3ce949618639a85d6b713df5732685e8b96a.js:9:342294
[10:56:19:0531]  INFO: clientLogger: Listen only media played successfully https://etc.bigbluebutton.org/html5client/d34c3ce949618639a85d6b713df5732685e8b96a.js?meteor_js_resource=true:125:1138810
```
Are the logs in the image just that @capilkey? I'm guessing it falls back to SIP.js for another reason other than autoplay. And then it hits an autoplay issue with SIP.js cause it wasn't hooked up for detecting autoplay errors.

**Comment by antobinary** [comment]
Addressed in #8357
